includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -582,6 +582,17 @@ public function javascript_params() {
 		$stripe_params['isAmazonPayEnabled']                = $express_checkout_helper->is_amazon_pay_enabled();
 		$stripe_params['isLinkEnabled']                     = $express_checkout_helper->is_link_enabled();
 
+		if ( $this->testmode ) {
+			/**
+			 * Filters whether the Stripe Developer Widget should be shown in the UI.
+			 * Only available in test mode.
+			 */
+			$show_stripe_developer_widget = (bool) apply_filters( 'wc_stripe_show_stripe_developer_widget', false );
```

PHPMD (2.15.0)

[warning] Avoid excessively long variable names like $show_stripe_developer_widget. Keep variable name length under 20. (undefined)

(LongVariable)